components/change_url_modal/index.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getCurrentTeamUrl} from 'mattermost-redux/selectors/entities/teams';
```

Comment:
Hmmm, unfortunately `getCurrentTeamUrl` is not working as expected. Refer to similar implementation here https://github.com/mattermost/mattermost-webapp/blob/master/components/get_post_link_modal/index.js

Reply:
@saturninoabril is this because the URL says `undefined` at the beginning. I believe its broken even there.

Reply:
Oops nvm I see what you mean!

Reply:
Fixed in my latest commit